packages/loading/src/createLoadingComponent.ts

```diff
@@ -1,5 +1,6 @@
-import { createVNode, h, reactive, ref, render, toRefs, Transition, VNode, vShow, withCtx, withDirectives } from 'vue'
+import { createVNode, h, reactive, ref, render, toRefs, Transition, VNode, vShow, withCtx, withDirectives, defineComponent } from 'vue'
 import { removeClass } from '@element-plus/utils/dom'
+import Empty from '../../empty/src/index.vue'
```

Comment:
Can't we use global import here?

```suggestion
import Empty from '@element-plus/empty'
```